src/Samples/CodeEditorContribution/CodeEditorContribution.ts

```diff
@@ -0,0 +1,59 @@
+import * as SDK from "azure-devops-extension-sdk";
+import { ICodeEditorContribution, ICodeEditorContributionEndpoints } from "./Contribution";
+
+SDK.init({

```

Comment:
I would keep this simpler and just call register, then init:

    SDK.register("my-code-editor-contribution", () => {
        ...
    });
    SDK.init();

Kind of a nit, but we want the extension sample to follow our recommended approach.